### PR TITLE
[WIP] Add documentation for provider certification tool

### PR DESCRIPTION
**Pull request by bostrt**
This work is for SPLAT-512. Adding documentation for provider certification tool CLI. I'm keeping it in WIP while we fill out a few more sections of content.

### CODE REVIEW

File: docs/certification/index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+### Check status
+
+```sh
+openshift-provider-cert status 
+openshift-provider-cert status -w # Keep watch open until completion
+```
+
+
+### Submit the artifacts
```
Comment (mtulio) [inline docs/certification/index.md:75]: Maybe changing to:
```suggestion
### Retrieve the results
```

**Review by mtulio — COMMENTED** [review]
Robert, I am thinking to deliver those sub-pages (under `Certification`):
- Overview
- Requirements / Prerequisites (is it includes installation?)
- Run (and follow status)
- Collect and Inspect results
  - Troubleshooting Guide
- Submitting the Results
- References

Following the structure of Gitbook / https://redhat-connect.gitbook.io/certification-guides/

Or more  nested like [CSI Certification](https://redhat-connect.gitbook.io/openshift-badges/badges/container-storage-interface-csi-1/workflow/running-the-csi-tests):
- Overview
- Requirements
(sub-pages of each requirement)
- Workflow
(sub-pages of each the functions)

Wdyt @bostrt @rvanderp3 ?

**Comment by elmiko** [comment]
this is really cool to see!

**Comment by bostrt** [comment]
@mtulio I'm not sure I would like to the pages split out since we wouldn't have good ctrl+f support and the other sections in this infrastructure provider doc don't follow that pattern. I would hope that this doc remains pretty small so we wouldn't need multiple pages either.

Here's what the rendered page looks like now which has the sections on the left. Do you think that would be a good compromise? 

Screenshot from 2022-06-08 10-49-35

**Comment by bostrt** [comment]
> i do wonder if it's appropriate to put the certification text next to the provider changes. we had talked about creating a new top level entry to the site and i'm wondering when we might start to consider that?

We're okay with moving these docs elsewhere if you think that would be best! I understand how they might feel outside the scope of provider code changes.

One requirement for us would be for someone to be able to make their way from the provider changes document to certification (just via a link with a short description).

We've also considered adding these docs to https://redhat-connect.gitbook.io/certification-guides/ which might make more sense.

**Comment by elmiko** [comment]
> We're okay with moving these docs elsewhere if you think that would be best! I understand how they might feel outside the scope of provider code changes.

i think to start with we are probably fine having them here, i just want to make sure that eventually they land somewhere where a partner would naturally expect them. in the future i imagine we will have a section about how a partner can supplement ocp without needing to change code, i would hope that we have the certification stuff linked there as well.

> One requirement for us would be for someone to be able to make their way from the provider changes document to certification (just via a link with a short description).
> 

+1

> We've also considered adding these docs to https://redhat-connect.gitbook.io/certification-guides/ which might make more sense.

it makes sense to have the cert stuff with the onboarding docs, i'm definitely sensitive the notion of spreading these docs out too much.

**Comment by mtulio** [comment]
> @mtulio I'm not sure I would like to the pages split out since we wouldn't have good ctrl+f support and the other sections in this infrastructure provider doc don't follow that pattern. I would hope that this doc remains pretty small so we wouldn't need multiple pages either.

+1 Completely agree. It's a good benefit to keep on a single page.

> Here's what the rendered page looks like now which has the sections on the left. Do you think that would be a good compromise?
> 

@bostrt sgtm that rendered page.

**Comment by bostrt** [comment]
After meeting with the team, we'll keep these docs in https://github.com/redhat-openshift-ecosystem/provider-certification-tool and simply link to them from docs.providers.openshift.org.

**Comment by bostrt** [comment]
https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/6

**Pull request closed without merging**